[
  {
    "name": "jupyter-core",
    "version": "4.11.2",
    "vulns": [
      {
        "id": "PYSEC-2022-42974",
        "fix_versions": [
          "5.0.0rc0"
        ]
      }
    ]
  }
]